docs/pages/auto-discovery/databases.mdx

```diff
@@ -26,15 +26,7 @@ discover AWS-hosted databases automatically.
 
 ## Step 1/4. Generate a join token
 
-The Discovery Service requires a valid join token to connect to the cluster.
-
-Generate a join token by running the following command against your Teleport
-Auth Service and save it in `/tmp/token` on the host that will run the
-Discovery Service.
-
-```code
-$ tctl tokens add --type=discovery
-```
+(!docs/pages/includes/tctl-token.mdx serviceName="Discovery" tokenType="discovery" tokenFile="/tmp/token" !)
```

Comment:
nit: does not need `tokenFile`?

Reply:
oh, you're right. I like using explicit args for partials to make it clear what the surrounding context should refer to. I'm actually going to just remove the default and make all the other includes specify `tokenFile`